app/src/main/res/drawable/rounded_image_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="@color/colorPrimary" />
```

Comment:
Use the widget's theme color as a default instead of always blue, or pick something which doesn't look out of place in a dynamic color theme (which is very hard and why theme colors exist, see main comment). The [widget's background](https://github.com/home-assistant/android/blob/2279d5d7dfb6151a0b4c3ac4ccdb60f251bc2056/app/src/main/res/drawable/widget_button_background.xml#L4) is an example of how they are used in another file.

```suggestion
    <solid android:color="?colorWidgetButton" />
```